### PR TITLE
Change colorSpread color-picker and other small fixes

### PR DESCRIPTION
**Pull request by AleziaKurdis**
Replace the color-picker of the attribute **particle.colorSpread** for a color vec3
To do this, a new type of input field has been added "**vec3rgb**" to support the "red, green, blue" vector (Non-color-picker)
Some **CSS** have been adjusted for the new field type '**vec3rgb**'.
This addresses issue **#593**

Fix also the broken particle texture preview that, as for '**atp:/**', doesn't support '**file:/**'. 
Needed for issue **#534**.

Revisit the tooltips for:
- **colorSpread**
- **localAngularVelocity** (Set the correct unit 'deg/s'. This addresses issue **#530**)
- **damping**
- **angularDamping**

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-a70a307.tar_f3a6fa52.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-a70a307_f3a6fa52.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-a70a307.tar_f3a6fa52.Z

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-a70a307_f3a6fa52.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-a70a307_f3a6fa52.exe

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-65dde7c_d31f161d.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-65dde7c.tar_d31f161d.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-65dde7c.tar_d31f161d.Z

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-65dde7c_d31f161d.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-65dde7c_d31f161d.exe

**Comment by AleziaKurdis** [comment]
To be tested:
1- Tooltip on the field: Agular Velocity (Create App > "Physic" tab) The unit in the text has been change for deg/s (instead of rad/s)
2- Deprecated Field "Acceleration" (Create App > "Physic" tab)  has been removed. (it was a "read only" field, in addition of being deprecated.)
3-The color-picker of the attribute "particle.colorSpread" has been changed for a color vec3 (Create App "Particle entity"> "Particle Color" tab)
    - Test that it can save and load values.
    - There is no changed on the functional side, only the wrong type of input has been fixed.
The way it works: if you set all the 3 particle color to gray (rgb 128,128,128)
if you set the particle.colorSpread to 128, 64, 32
Each particle will get a color randomly calculated from inside those intervals:  
Red (0 to 256, because it will be + or - 128 around the 128)
Green (64 to 192, because it will be + or - 64 around the 128)
Blue (96 to 160, because it will be + or - 32 around the 128)
So you should see the particle lightly colored with colors from this 3 composite interval.

4- For particle entity, the preview would display "Not Available" if you are using a url starting with "file://"
But this won't be really used since we decided that the default particle won't be a local file.


I hope this would help the QA. Ping me directly if this is still obscure.

File: scripts/system/create/entityProperties/html/js/entityProperties.js [hunk 1]
```diff
@@ -2856,7 +2890,7 @@ function createTextureProperty(property, elProperty) {
 
     let imageLoad = function(url) {
         elDiv.style.display = null;
-        if (url.slice(0, 5).toLowerCase() === "atp:/") {
+        if (url.slice(0, 5).toLowerCase() === "atp:/" || url.slice(0, 6).toLowerCase() === "file:/") {
```
Comment (ctrlaltdavid) [inline scripts/system/create/entityProperties/html/js/entityProperties.js:2893]: Is this change needed still?

Comment (AleziaKurdis) [reply scripts/system/create/entityProperties/html/js/entityProperties.js:2893]: Yes. This is possible to use such "file://" URL (it works)
despite the fact that it's not something we would like to do promotion of.
So the fix is still relevant, to not have a broken preview.
Such relative URLs can't be resolved bu the web component (as for "atp:/") so this code is to address that.

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-fe6b30f.tar_8d217d21.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-fe6b30f.tar_8d217d21.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-fe6b30f_8d217d21.sh

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-fe6b30f_8d217d21.exe

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR606/Vircadia-Alpha-PR606-fe6b30f_8d217d21.dmg
